Problem after generating app on Windows

<!--
- Please follow the issue template below for bug reports.
- If you have a support request rather than a bug, please use [Stack Overflow](http://stackoverflow.com/questions/tagged/jhipster) with the JHipster tag.
- For bug reports it is mandatory to run the command `jhipster info` in your project's root folder, and paste the result here.
- Tickets opened without any of these pieces of information will be **closed** without any explanation.
-->

##### **Overview of the issue**

Fails to run application on Windows 10 machine but on docker inside a linux container it runs without any issue. 

When running `dotnet run --verbosity normal --project ./src/Jhipster/Jhipster.csproj` as the generator suggests, the following error is shown:

npm webpack:build

Usage: npm <command>

where <command> is one of:
    access, adduser, audit, bin, bugs, c, cache, ci, cit,
    clean-install, clean-install-test, completion, config,
    create, ddp, dedupe, deprecate, dist-tag, docs, doctor,
    edit, explore, fund, get, help, help-search, hook, i, init,
    install, install-ci-test, install-test, it, link, list, ln,
    login, logout, ls, org, outdated, owner, pack, ping, prefix,
    profile, prune, publish, rb, rebuild, repo, restart, root,
    run, run-script, s, se, search, set, shrinkwrap, star,
    stars, start, stop, t, team, test, token, tst, un,
    uninstall, unpublish, unstar, up, update, v, version, view,
    whoami

npm <command> -h  quick help on <command>
npm -l            display full usage info
npm help <term>   search for help on <term>
npm help npm      involved overview

Specify configs in the ini-formatted file:
    C:\Users\default.DESKTOP-NO1C5HC\.npmrc
or on the command line via: npm <command> --key value
Config info can be viewed via: npm help config

npm@6.14.4 C:\Programas\nvm\v12.16.3\node_modules\npm


C:\Users\default.DESKTOP-NO1C5HC\projetos\jhipster-app-core-030\src\Jhipster\ClientApp>npm run webpack:build 

> jhipster@0.0.1-SNAPSHOT webpack:build C:\Users\default.DESKTOP-NO1C5HC\projetos\jhipster-app-core-030\src\Jhipster\ClientApp
> npm run cleanup && npm run webpack:build:main


> jhipster@0.0.1-SNAPSHOT cleanup C:\Users\default.DESKTOP-NO1C5HC\projetos\jhipster-app-core-030\src\Jhipster\ClientApp
> rimraf bin/aot && rimraf dist/*


> jhipster@0.0.1-SNAPSHOT webpack:build:main C:\Users\default.DESKTOP-NO1C5HC\projetos\jhipster-app-core-030\src\Jhipster\ClientApp
> npm run webpack -- --config webpack/webpack.dev.js --env.stats=minimal


> jhipster@0.0.1-SNAPSHOT webpack C:\Users\default.DESKTOP-NO1C5HC\projetos\jhipster-app-core-030\src\Jhipster\ClientApp
> node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js "--config" "webpack/webpack.dev.js" "--env.stats=minimal"

Webpack: Starting ...

  √ Compile modules
  √ Build modules
  √ Optimize modules
  √ Emit files

Webpack: Finished after 26.645 seconds.



 ERROR  Failed to compile with 1 errors                                                                                                        21:35:04
 error 


   457 modules

ERROR in   Error: Child compilation failed:
  Module parse failed: Unexpected character '@' (1:0)
  You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/con  cepts#loaders
  > @keyframes lds-pacman-1 {
  |   0% {
  |     -webkit-transform: rotate(0deg);:
  SyntaxError: Unexpected character '@' (1:0)

  - compiler.js:79
    [ClientApp]/[html-webpack-plugin]/lib/compiler.js:79:16

  - Compiler.js:343
    [ClientApp]/[webpack]/lib/Compiler.js:343:11

  - Compiler.js:681
    [ClientApp]/[webpack]/lib/Compiler.js:681:15


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [ClientApp]/[tapable]/lib/Hook.js:154:20

  - Compiler.js:678
    [ClientApp]/[webpack]/lib/Compiler.js:678:31


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [ClientApp]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1423
    [ClientApp]/[webpack]/lib/Compilation.js:1423:35


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [ClientApp]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1414
    [ClientApp]/[webpack]/lib/Compilation.js:1414:32


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [ClientApp]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1409
    [ClientApp]/[webpack]/lib/Compilation.js:1409:36


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [ClientApp]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1405
    [ClientApp]/[webpack]/lib/Compilation.js:1405:32


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [ClientApp]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1342 Compilation.seal
    [ClientApp]/[webpack]/lib/Compilation.js:1342:27

  - Compiler.js:675
    [ClientApp]/[webpack]/lib/Compiler.js:675:18

  - Compilation.js:1261
    [ClientApp]/[webpack]/lib/Compilation.js:1261:4


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [ClientApp]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1253 Compilation.finish
    [ClientApp]/[webpack]/lib/Compilation.js:1253:28

  - Compiler.js:672
    [ClientApp]/[webpack]/lib/Compiler.js:672:17


  - Compilation.js:1185
    [ClientApp]/[webpack]/lib/Compilation.js:1185:12

  - Compilation.js:1097
    [ClientApp]/[webpack]/lib/Compilation.js:1097:9

  - task_queues.js:79 processTicksAndRejections
    internal/process/task_queues.js:79:11


npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! jhipster@0.0.1-SNAPSHOT webpack: `node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js "--config" "webpack/webpack.dev.js" "--env.stats=minimal"`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the jhipster@0.0.1-SNAPSHOT webpack script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\default.DESKTOP-NO1C5HC\AppData\Roaming\npm-cache\_logs\2020-05-07T00_35_04_780Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! jhipster@0.0.1-SNAPSHOT webpack:build:main: `npm run webpack -- --config webpack/webpack.dev.js --env.stats=minimal`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the jhipster@0.0.1-SNAPSHOT webpack:build:main script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\default.DESKTOP-NO1C5HC\AppData\Roaming\npm-cache\_logs\2020-05-07T00_35_04_824Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! jhipster@0.0.1-SNAPSHOT webpack:build: `npm run cleanup && npm run webpack:build:main`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the jhipster@0.0.1-SNAPSHOT webpack:build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\default.DESKTOP-NO1C5HC\AppData\Roaming\npm-cache\_logs\2020-05-07T00_35_04_858Z-debug.log

<!-- Explain the bug, if an error is being thrown a stack trace helps -->

##### **Motivation for or Use Case**

<!-- Explain why this is a bug for you -->
This is important because most .net developers have familiarity with Windows as a developer machine

##### **Reproduce the error**

<!-- For bug reports, an unambiguous set of steps to reproduce the error -->
Link blueprint globally
`cd dotnetcore`

`npm link`

`cd ..`

`mkdir my-app && cd my-app`

`npm link generator-jhipster-dotnetcore`

Run `jhipster --blueprints dotnetcore` and all files are generated.

After that try to run the application with 
`dotnet run --verbosity normal --project ./src/Jhipster/Jhipster.csproj`
and error is shown on webpack build.

##### **Related issues**

<!-- Has a similar issue been reported before? Please search both closed & open issues -->

##### **Suggest a Fix**

<!-- If you can't fix the bug yourself, perhaps you can point to what might be
  causing the problem (line of code or commit) -->
Seems to be related to the css files and the @ symbol not being interpreted by webpack but I'm not sure.

##### **JHipster Version(s)**

<!--
Which version of JHipster are you using, is it a regression?
-->
latest version of dotnet blueprint 

##### **JHipster configuration**

INFO! Using JHipster version installed globally
INFO! No custom sharedOptions found within blueprint: generator-jhipster-dotnetcore at C:/Users/default.DESKTOP-NO1C5HC/projetos/jhipster-app-core-030/node_modules/generator-jhipster-dotnetcore
INFO! Executing jhipster:info
INFO! Options: from-cli: true
##### **JHipster Version(s)**

```
C:\Users\default.DESKTOP-NO1C5HC\projetos\jhipster-app-core-030
`-- generator-jhipster-dotnetcore@0.3.0 -> C:\Users\default.DESKTOP-NO1C5HC\projetos\generator-jhipster-dotnetcore
  `-- generator-jhipster@6.8.0  extraneous (git://github.com/jhipster/generator-jhipster.git#f389028b96ca650ed5ce773c9fe959e6a38b7e0d)

```


##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**


<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "namespace": "MyCompany",
    "databaseType": "sqllite",
    "authenticationType": "jwt",
    "prodDatabaseType": "mysql",
    "jhipsterVersion": "6.8.0",
    "applicationType": "monolith",
    "baseName": "jhipster",
    "useSass": true,
    "clientPackageManager": "npm",
    "clientFramework": "angularX",
    "clientTheme": "none",
    "clientThemeVariant": "",
    "creationTimestamp": 1588807104239,
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": [
      {
        "name": "generator-jhipster-dotnetcore",
        "version": "0.3.0"
      }
    ],
    "enableTranslation": false,
    "blueprints": [
      {
        "name": "generator-jhipster-dotnetcore",
        "version": "0.3.0"
      }
    ]
  }
}

</pre>
</details>


##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>


##### **Environment and Tools**

java version "1.8.0_221"
Java(TM) SE Runtime Environment (build 1.8.0_221-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.221-b11, mixed mode)

git version 2.9.0.windows.1

node: v12.16.3

npm: 6.14.4

Docker version 19.03.8, build afacb8b

docker-compose version 1.25.5, build 8a1c60f6

INFO! Congratulations, JHipster execution is complete!

<!--
To provide all information we need, you should run `jhipster info` in the project root folder, and
copy/paste the result here.
The `.yo-rc.json` file generated in the root folder is mandatory for bug reports. This will help us to replicate the scenario.
You should remove any sensitive information like the rememberMe key or the jwtSecretKey key.
---
If you have a JDL please wrap it in below structure
  <details>
  <summary>JDL definitions</summary>
  <pre>
     JDL content here
  </pre>
  </details>
-->

##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<!--
If the error is during an entity creation or associated with a specific entity.
If you are using JDL, please share that configuration as well.
-->

##### **Browsers and Operating System**
Only on Windows 10 (tested on docker container on ubuntu linux 18.08 and it works)

<!-- What OS are you on? is this a problem with all browsers or only IE8? -->

-   [x] Checking this box is mandatory (this is just to show you read everything)

<!-- Love JHipster? Please consider supporting our collective:
👉  https://opencollective.com/generator-jhipster/donate -->

--- a/generators/client/files-common.js
+++ b/generators/client/files-common.js
@@ -35,6 +35,12 @@ function updateWebpackCommonJs() {
         "",
         false
     );
+    this.replaceContent(
+        `${SERVER_SRC_DIR}${this.mainClientDir}/webpack/webpack.common.js`,
+        "utils.root('src/main/webapp/index.html')",
+        "utils.root('src/index.html')",
+        false
+    );
 }
 
 function updateWebpackDevJs() {

